policyengine_us/variables/gov/states/ri/tax/income/adjusted_gross_income/taxable_retirement_income/ri_taxable_retirement_income_subtraction.py

```diff
@@ -0,0 +1,18 @@
+from policyengine_us.model_api import *
+
+
+class ri_taxable_retirement_income_subtraction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Rhode Island Taxable retirement income subtraction"
+    unit = USD
+    definition_period = YEAR
+    reference = "http://webserver.rilin.state.ri.us/Statutes/title44/44-30/44-30-12.HTM"
+    defined_for = "ri_taxable_retirement_income_subtraction_eligible"
+
+    def formula(tax_unit, period, parameters):
+        taxable_pension = tax_unit.members("taxable_pension_income", period)
+        p = parameters(
+            period
+        ).gov.states.ri.tax.income.adjusted_gross_income.subtractions.taxable_retirement_income
+        return min_(taxable_pension, p.cap)
```

Comment:
need a tax_unit.sum

Reply:
> need a tax_unit.sum

Question: check the code and I think here the cap of $15,000 is for each pension plan - what is the relationship between plan, person and tax unit? Do we consider the whole tax unit within only one pension plan?

ref: http://webserver.rilin.state.ri.us/Statutes/title44/44-30/44-30-12.HTM (c) (9), key word "taxable retirement income"